language-server/src/features/workspace.js

```diff
@@ -62,7 +76,9 @@ export default {
       reporter.begin("JSON Schema: Indexing workspace");
 
       // Re/validate all schemas
-      for await (const uri of workspaceSchemas()) {
+      const result = await getDocumentSettings(connection);
+      const schemaFilePatterns = result.schemaFilePatterns;
```

Comment:
I think this variable should have a better name. How about `settings`?

Reply:
yes, `settings `would be a better name.